docs/js/visualization.js

```diff
@@ -456,16 +480,17 @@ const get_samples = () => {
             scatterPlotEnter.merge(scatterPlotUpdate)
         }
 
-        function getSimilarityScore() {
+        function getSimilarityScores() {
             let x1 = similaritySelections[0][0];
             let x2 = similaritySelections[1][0];
             let y1 = similaritySelections[0][1];
             let y2 = similaritySelections[1][1];
-            return ((x1*x2)+(y1*y2)) / Math.sqrt(((x1**2) + (y1**2)) * ((x2**2) + (y2**2)));
+            let cosineScore = ((x1*x2)+(y1*y2)) / Math.sqrt(((x1**2) + (y1**2)) * ((x2**2) + (y2**2)));
+            let euclideanScore = Math.sqrt((x2 - x1)**2 + (y2 - y1)**2);
+            return {"cosine": Math.floor(cosineScore * 10000) / 100, "euclidean": Math.floor(euclideanScore * 10000) / 100};
```

Comment:
Modify the cosineScore calculation to reflect degrees rather than percentages

Reply:
done.